include/boost/url/impl/pct_encoding.hpp

```diff
@@ -378,10 +305,10 @@ template<
 std::basic_string<char,
     std::char_traits<char>,
         Allocator>
-pct_encode(
+pct_encode_to_string(
     string_view s,
+    CharSet const& allowed,
     pct_encode_opts const& opt,
-    CharSet const& cs,
     Allocator const& a)
```

Comment:
no default Allocator?

Reply:
The default parameter is in the declaration. This is the implementation file.